### DISCUSSION TITLE
Send email X days before expire

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#347] [Feature Requests]
As a user, I want to have the ability to send notification email X days before expires. It will helpful to notify administrator that there will be post expiring soon.

**Comment by kavicka** [comment]
Hello, is this feature available now? Also, would it be possible to hook on this action? I would like send notification as SMS and send email to a post author. I would handle this by myself, I just need to hook on this action.

**Reply by andergmartins** [reply]
@kavicka not available yet, sorry. You can upvote it (if didn't do that yet).
Regarding the question about the hook, I answered adding this doc page: https://publishpress.com/knowledge-base/technical-details/#how-to-hook-into-action-schedule-posts

**Comment by andergmartins** [comment]
This is possible using the new Action Workflows feature in the Pro version. We just need to implement #843 to allow triggering a custom action hook to run custom code.
